ios/Classes/NativeBridge/Handlers/DeregisterUserHandler.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+import OneginiSDKiOS
+import OneginiCrypto
+
+protocol DeregisterUserHandlerProtocol: AnyObject {
+    func deregister(_ profileId: String, completion: @escaping (SdkError?) -> Void)
+}
+
+class DeregisterUserHandler: DeregisterUserHandlerProtocol {
+    func deregister(_ profileId: String, completion: @escaping (SdkError?) -> Void){
+        let userClient = ONGUserClient.sharedInstance()
+        
+        guard let profile = ONGUserClient.sharedInstance().userProfiles().first(where: { $0.profileId == profileId }) else {
+            return completion(SdkError(.userProfileDoesNotExist))
```

Comment:
Looks like you changed it everywhre, didn't you? @MichielVrins haven't we the same discussion? I'm having déjà vu :)

Reply:
We had the same discussion @szymon-onewelcome I believe the result of the discussion was that it actually does do the same as 
```
completion(SdkError(.userProfileDoesNotExist))
return
```
But we chose to do it on 2 lines instead of 1 line as that is the way it is usually done in swift.

Reply:
clear, updated